dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/ApplicationInstallInfo.java

```diff
@@ -81,6 +84,12 @@ public boolean getUseJandex() {
         return _config.getUseJandex();
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String getAnnotationScanLibrary() {
```

Comment:
Since this is the new SPI, there should be a beta check done here if it is called by the user.